index.bs

```diff
@@ -119,6 +119,19 @@ To <dfn>Construct an AmbientLightSensor Object</dfn> the user agent must invoke
 The <a attribute for="AmbientLightSensor">illuminance</a> attribute of the {{AmbientLightSensor}}
 interface represents the <a>current light level</a>.
 
+Use Cases and Requirements {#usecases-requirements}
+=========
+
+1. A Web application gradually updates document style based on light level changes.
```

Comment:
Enum values don't allow to create 'designer' specific ranges or control style changes gradually. ALS sensors on mobile devices could provide values in 0-43000 Lux range. Maybe some pages could provide high-contrast monochrome style for 'direct-sun' conditions. It would be difficult to define 3 enum values for 0-43000 range, that would satisfy everyone.